superset/tasks/scheduler.py

```diff
@@ -47,9 +48,15 @@ def scheduler() -> None:
         return
     with session_scope(nullpool=True) as session:
         active_schedules = ReportScheduleDAO.find_active(session)
+        triggered_at = (
+            datetime.fromisoformat(scheduler.request.expires)
+            - app.config["CELERY_BEAT_SCHEDULER_EXPIRES"]
```

Comment:
Hi I've been checking this code, how is this solving the issue, it select `datetime.utcnow()` because `datetime.fromisoformat(scheduler.request.expires) - app.config["CELERY_BEAT_SCHEDULER_EXPIRES"] = 0`
Can you help me understand @jfrag1 ??

Reply:
@piyushdatazip I'm not sure I fully understand what you're asking, but `datetime.fromisoformat(scheduler.request.expires)` is a `datetime`, and `app.config["CELERY_BEAT_SCHEDULER_EXPIRES"]` is a `timedelta`, so the result of this subtraction is a `datetime`, not 0.

Reply:
Thanks @jfrag1 for replying, my bad I've understood it wrongly at first instance